Add Knowledge Base mention to IaC Library Guide

pete0emerson opened the pull request.

brikis98 reviewed (commented):
Thanks Pete! A few other places to update:

1. https://gruntwork.io/support/
2. https://gruntwork.io/how-it-works/#6-get-expert-support
3. https://gruntwork.io/pricing/: specifically, the "Community Support" mouse-over.
4. https://gruntwork.io/checkout/?subscription-type=standard: again, the "Community Support" mouse-over.

pete0emerson commented:
>     1. https://gruntwork.io/support/

Addressed in 020b7b6b.

>     2. https://gruntwork.io/how-it-works/#6-get-expert-support

Addressed in 6031d984.

>     3. https://gruntwork.io/pricing/: specifically, the "Community Support" mouse-over.

Addressed in 8ebf614e.

>     4. https://gruntwork.io/checkout/?subscription-type=standard: again, the "Community Support" mouse-over.

Addressed in 00f8b8fd.

brikis98 reviewed (approved):
LGTM!
